pkgs/by-name/cr/crab-hole/package.nix

```
@@ -0,0 +1,30 @@
+{
+  lib,
+  fetchFromGitHub,
+  rustPlatform,
+}:
+
+rustPlatform.buildRustPackage {
+  pname = "crab-hole";
+  version = "0.1.10";
+
+  src = fetchFromGitHub {
+    owner = "LuckyTurtleDev";
+    repo = "crab-hole";
+    rev = "eae420c94679d17db7f6bf498eec258a792cdd18";
```

With this you should be able to use `nix run nixplgs#nix-update -- --commit crab-hole` to automatically update and commit the nix expression when the new version of `crab-hole` is tagged.
```suggestion
    rev = "refs/tags/${version}";
```
But you'll need a `rec` after `rustPlatform.buildRustPackage`